code/modules/mob/living/silicon/ai/life.dm

```diff
@@ -171,4 +174,9 @@
 
 /mob/living/silicon/ai/rejuvenate()
 	..()
-	add_ai_verbs(src)
\ No newline at end of file
+	add_ai_verbs(src)
+
+/mob/living/silicon/ai/proc/process_nations()

```

Comment:
Why is this a thing here and in human life as well; this shouldn't be a duplicate proc.